concent_api/core/tests/test_core_views.py

```diff
@@ -477,22 +482,30 @@ def test_send_task_to_compute_without_public_key_should_return_http_400(self):
     def test_send_task_to_compute_with_public_key_with_wrong_length_should_return_http_400(self):
         assert StoredMessage.objects.count() == 0
 
-        setattr(self.task_to_compute, 'requestor_public_key', getattr(self.task_to_compute, 'requestor_public_key')[:-1])
+        setattr(self.report_computed_task.task_to_compute, 'requestor_public_key', getattr(self.task_to_compute, 'requestor_public_key')[:-1])
 
         response = self._send_force_report_computed_task()
 
         self._test_400_response(
             response,
-            error_message=ERROR_IN_GOLEM_MESSAGE,
+            error_code=ErrorCode.MESSAGE_VALUE_WRONG_LENGTH,
         )
         self._assert_stored_message_counter_not_increased()
         self._assert_client_count_is_equal(0)
 
     @freeze_time("2017-11-17 10:00:00")
     def test_error_in_golem_messages_should_be_logged_and_return_http_400_with_the_same_uuid(self):
-        self.task_to_compute.requestor_public_key = self.task_to_compute.requestor_public_key[:-1]
+        dumped_message = dump(
+            msg=message.concents.ForceReportComputedTask(
+                report_computed_task=message.tasks.ReportComputedTask(
+                    task_to_compute=self.task_to_compute
+                ),
+            ),
+            privkey=self.PROVIDER_PRIVATE_KEY,
+            pubkey=settings.CONCENT_PUBLIC_KEY,
+        )
         with mock.patch('core.decorators.log') as log:
-            response = self._send_force_report_computed_task()
+            response = self._send_force_report_computed_task(serialized_force_report_computed_task=dumped_message[:-1])
```

Comment:
please move this slicing (`[-:1]` to the variable definition - otherwise the name is wrong ;)

Reply:
Changed :)